### PR TITLE
saltsshpush methods: lack of reverse tunnel

### PR DESCRIPTION
**Pull request by keichwa**
**For rendered output,** see link in https://github.com/SUSE/spacewalk/issues/25534

- https://github.com/SUSE/spacewalk/issues/25534
- https://github.com/uyuni-project/uyuni-docs/issues/1892

### CODE REVIEW

File: modules/client-configuration/pages/contact-methods-saltssh-tunnel.adoc [hunk 1]
```diff
@@ -34,18 +40,21 @@ To overwrite it, you can add a custom port numbers greater than 1024 to [path]``
...
-[systemitem]``tomcat.service``, and
-[systemitem]``taskomatic.service``.  To cover all needed services, it
-is the best to restart [command]``spacewalk-service`` as root:
+After changing this configuration parameter you must restart services such as [systemitem]``salt-secrets-config.service``, [systemitem]``tomcat.service``, and
+[systemitem]``taskomatic.service``.
+To cover all needed services, it is the best to restart the {productname} Server as root:
 
 ----
-spacewalk-service restart
+mgradm restart
```
Comment (mcalmer) [inline modules/client-configuration/pages/contact-methods-saltssh-tunnel.adoc:48]: Above you said run inside of the container. Inside of the container `spacewalk-service restart` is correct.
`mgradm restart` will restart the whole container and need to be run outside.

Maybe you describe both like:

> Inside the container run `spacewalk-service restart` or as alternative you can also restart the whole container when we run on the host `mgradm restart`.

... or similar.

File: modules/client-configuration/pages/contact-methods-saltssh-tunnel.adoc [hunk 2]
```diff
@@ -26,6 +26,12 @@ In other words, access is only possible if the session is initiated by the serve
 All package managing operations on the client can be performed from the server side only.
 ====
 
+
+[IMPORTANT]
+====
+Use [literal]``mgrctl term`` before running steps inside the server container.
+====
```
Comment (mcalmer) [inline modules/client-configuration/pages/contact-methods-saltssh-tunnel.adoc:33]: The key is to say which steps need to run inside of the container and which not. This sentence expect that the customer know that need to be executed where.

File: modules/client-configuration/pages/contact-methods-saltssh-tunnel.adoc [hunk 3]
```diff
@@ -34,18 +40,21 @@ To overwrite it, you can add a custom port numbers greater than 1024 to [path]``
...
-spacewalk-service restart
+mgradm restart
 ----
 
 
 
+[IMPORTANT]
+====
+Use [literal]``mgrctl term`` before running steps inside the server container.
+====
```
Comment (mcalmer) [inline modules/client-configuration/pages/contact-methods-saltssh-tunnel.adoc:56]: The key is to say which steps need to run inside of the container and which not. This sentence expect that the customer know that need to be executed where.

And here the "sudo" configuration happens on the client. This is not in a container

File: modules/client-configuration/pages/contact-methods-saltssh-tunnel.adoc [hunk 4]
```diff
@@ -77,18 +86,15 @@ ssh_push_sudo_user = <user>
...
-[systemitem]``tomcat.service``, and
-[systemitem]``taskomatic.service``.  To cover all needed services, it
-is the best to restart [command]``spacewalk-service`` as root:
+After changing this configuration parameter you must restart services such as [systemitem]``salt-secrets-config.service``, [systemitem]``tomcat.service``, and
+[systemitem]``taskomatic.service``.
+To cover all needed services, it is the best to restart the {productname} Server as root:
 
 ----
-spacewalk-service restart
+mgradm restart
```
Comment (mcalmer) [inline modules/client-configuration/pages/contact-methods-saltssh-tunnel.adoc:94]: Same as above

File: modules/client-configuration/pages/contact-methods-saltssh.adoc [hunk 5]
```diff
@@ -75,6 +80,10 @@ These parameters are configured in menu:Systems[Bootstrapping].
 You can also configure persistent parameters that are used system-wide, including the sudo user to access the system as an unprivileged user instead of as root.
 
 
+[IMPORTANT]
+====
+Use [literal]``mgrctl term`` before running steps inside the server container.
+====
```
Comment (mcalmer) [inline modules/client-configuration/pages/contact-methods-saltssh.adoc:86]: Same as above. The sudo configuration is not on the container.
We need to explain which steps need to run inside the container and which outside or on a complete different host

**Review by mcalmer — CHANGES_REQUESTED** [review]

File: modules/client-configuration/pages/contact-methods-saltssh-tunnel.adoc (hunk 2 above)
Comment (keichwa) [reply modules/client-configuration/pages/contact-methods-saltssh-tunnel.adoc:33]: yes, we still must find a good way to communicate it.

File: modules/client-configuration/pages/contact-methods-saltssh-tunnel.adoc (hunk 1 above)
Comment (keichwa) [reply modules/client-configuration/pages/contact-methods-saltssh-tunnel.adoc:49]: yes, I changed it for clarity.

File: modules/client-configuration/pages/contact-methods-saltssh.adoc (hunk 5 above)
Comment (keichwa) [reply modules/client-configuration/pages/contact-methods-saltssh.adoc:86]: yes, but within the procedure, we clarify it step-wise ("On each client system" vs. "On the {productname} Server" (now I use "Inside the {productname} Server container, ...").  But I agree that this still can confuse users.

**Comment by keichwa** [comment]
I'll also update the rendered output soon.

**Pull request merged**